AssetBundleManager/AssetBundleManager.cs

```diff
@@ -359,6 +359,48 @@ public void UnloadBundle(string bundleName, bool unloadAllLoadedObjects, bool fo
             }
         }
 
+        /// <summary>
+        ///     Returns if the Asset Bundle is already loaded
+        /// </summary>
+        public bool IsAssetBundleLoaded(string bundleName)
+        {
+            if (activeBundles.ContainsKey(bundleName)) {
+                return true;
+            } else {
+                return false;
+            }
+        }
+
+        /// <summary>
+        ///     Returns if the Asset Bundle is already loaded (out Bundle if loaded)
+        /// </summary>
+        public bool IsAssetBundleLoaded(string bundleName, out AssetBundle bundle)
```

Comment:
It's a minor thing, but could you rename this to TryGetAssetBundle?  That way it matches c# conventions that have similar functionality.